<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_AutoAlpha</class>
 <widget class="QDialog" name="Dialog_AutoAlpha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alpha Determination</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_19">
   <item>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>710</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_31">
       <item>
        <widget class="QScrollArea" name="scrollArea_image">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>626</width>
            <height>690</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLabel" name="label_imageWinSelection">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QwtPlot" name="qwtPlot_Skew">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>2</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>450</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_32">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QGroupBox" name="groupBox_14">
                <property name="title">
                 <string>Windows Selection</string>
                </property>
                <layout class="QFormLayout" name="formLayout_3">
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_33">
                   <item>
                    <widget class="QLabel" name="label_22">
                     <property name="text">
                      <string>P1x: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spinBox_PointP1_x">
                     <property name="maximum">
                      <number>9000</number>
                     </property>
                     <property name="singleStep">
                      <number>1</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout_34">
                   <item>
                    <widget class="QLabel" name="label_23">
                     <property name="text">
                      <string>P2x: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spinBox_PointP2_x">
                     <property name="maximum">
                      <number>9000</number>
                     </property>
                     <property name="singleStep">
                      <number>1</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_35">
                   <item>
                    <widget class="QLabel" name="label_24">
                     <property name="text">
                      <string>P1y: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spinBox_PointP1_y">
                     <property name="maximum">
                      <number>9000</number>
                     </property>
                     <property name="singleStep">
                      <number>1</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout_36">
                   <item>
                    <widget class="QLabel" name="label_25">
                     <property name="text">
                      <string>P2y: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spinBox_PointP2_y">
                     <property name="maximum">
                      <number>9000</number>
                     </property>
                     <property name="singleStep">
                      <number>1</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_saveSelection">
                <property name="text">
                 <string>Save Selection (Temp)</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item>
               <widget class="QGroupBox" name="groupBox_15">
                <property name="title">
                 <string>Alpha Estimation Parameters</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_16">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_37">
                   <item>
                    <spacer name="horizontalSpacer_11">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_26">
                     <property name="text">
                      <string>From:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="doubleSpinBox_AlphaFrom">
                     <property name="minimum">
                      <double>0.010000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>0.010000000000000</double>
                     </property>
                     <property name="value">
                      <double>0.100000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_38">
                   <item>
                    <spacer name="horizontalSpacer_12">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_27">
                     <property name="text">
                      <string>To:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="doubleSpinBox_AlphaTo">
                     <property name="minimum">
                      <double>0.100000000000000</double>
                     </property>
                     <property name="maximum">
                      <double>100.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>1.000000000000000</double>
                     </property>
                     <property name="value">
                      <double>10.000000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_39">
                   <item>
                    <spacer name="horizontalSpacer_13">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_28">
                     <property name="text">
                      <string>Step Size:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="doubleSpinBox_AlphaStepSize">
                     <property name="minimum">
                      <double>0.010000000000000</double>
                     </property>
                     <property name="maximum">
                      <double>10.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>0.010000000000000</double>
                     </property>
                     <property name="value">
                      <double>0.100000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_40">
                   <item>
                    <widget class="QLabel" name="label_29">
                     <property name="text">
                      <string>Number Slices:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLCDNumber" name="lcdNumber_NoSlice">
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 0);
border-color: rgb(0, 0, 127);</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_17">
              <item>
               <widget class="QGroupBox" name="groupBox_16">
                <property name="title">
                 <string>Alpha Estimation process</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_18">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_41">
                   <item>
                    <spacer name="horizontalSpacer_14">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_StartAlphaEstimation">
                     <property name="text">
                      <string>Start</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_42">
                   <item>
                    <widget class="QLabel" name="label_30">
                     <property name="text">
                      <string>Proposed Alpha:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="doubleSpinBox_AlphaResult">
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(85, 255, 0);</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_43">
           <item>
            <widget class="QGroupBox" name="groupBox_17">
             <property name="title">
              <string>Zoom Image</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_44">
              <item>
               <widget class="QComboBox" name="comboBox_ImageZoom">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>25%</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>50%</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>75%</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>100%</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>200 %</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_15">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_18">
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_45">
              <item>
               <widget class="QPushButton" name="pushButton_accept">
                <property name="text">
                 <string>Accept Alpha</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_9">
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_accept</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_AutoAlpha</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1121</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>1306</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_AutoAlpha</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1235</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>1295</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
